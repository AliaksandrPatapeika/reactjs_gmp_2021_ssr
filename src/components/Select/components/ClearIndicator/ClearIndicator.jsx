import React from 'react';
import {components} from 'react-select';

const ClearIndicator = (props) => (
  <components.ClearIndicator {...props}>
    <i className="fa fa-times" aria-hidden="true" />
  </components.ClearIndicator>
);

export default ClearIndicator;
