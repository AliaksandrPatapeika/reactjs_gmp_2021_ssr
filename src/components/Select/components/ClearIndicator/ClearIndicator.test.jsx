import {noop} from 'lodash';
import React from 'react';
import renderer from 'react-test-renderer';

import ClearIndicator from './ClearIndicator';

describe('ClearIndicator', () => {
  test('renders correctly', () => {
    const clearIndicator = renderer
      .create(<ClearIndicator getStyles={noop} cx={noop} />)
      .toJSON();

    expect(clearIndicator).toMatchSnapshot();
  });
});
