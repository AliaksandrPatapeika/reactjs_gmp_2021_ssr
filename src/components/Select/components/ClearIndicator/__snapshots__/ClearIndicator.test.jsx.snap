// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClearIndicator renders correctly 1`] = `
<div
  className="css-h0vbma-ClearIndicator"
>
  <i
    aria-hidden="true"
    className="fa fa-times"
  />
</div>
`;
